usaspending_api/download/v2/download_column_historical_lookups.py

```diff
@@ -1469,6 +1473,10 @@
         ),
         "federal_account": OrderedDict(
             [
+                (
+                    "last_reported_submission_period",
+                    "last_reported_submission_period",
```

Comment:
The current pattern is to set the "DB field name" value to null if it will be populated by an annotation

Reply:
Annotations work differently in the custom account download part of the site.  I spent 45 minutes trying to make them work like the other downloads and failed.  This is bigger than a breadbox and I would like to kick the can down the road so I can make progress on other tickets.

The code that I checked in works like other code for these types of downloads.